spec/support/features.rb

```diff
@@ -0,0 +1,18 @@
+require "capybara/rails"
+require "gds_api/test_helpers/content_store"
+
+module FeatureHelpers
+  include GdsApi::TestHelpers::ContentStore
+
+  def assert_bank_holiday_table(attrs)
```

Comment:
How were the minitest tests passing before this code was added? (I'm looking at commit 2130b7764c43144919e0900b4f08a8e528d1c96f). I suspect you've removed the `assert_bank_holiday_table` earlier in the history, and added it back in here, but it would be good if you removed the minitest version in the same commit as this one. It currently looks like you wrote this code.

Reply:
The minitest version of `assert_bank_holiday_table` is removed in 8e2313f52e98d7c59430f6172c81c5b02dad482a, which is the final commit on the branch.  Github often displays commits in the wrong order after rebasing.

I could put the removal in 2130b7764c43144919e0900b4f08a8e528d1c96f, but I'd also have to squash in e67f36e0b4b770a01a39169f96604e5c7c764c1e, as that uses the method too.

Reply:
Ah, makes sense. (For what it's worth, it looks like GitHub is rendering the order correctly.)

Happy to leave it 👍

Reply:
It is in the commit list for the branch, but not in the PR, where 8e2313f52e98d7c59430f6172c81c5b02dad482a is somewhere in the middle 😞 

<img width="686" alt="Screenshot 2020-04-23 at 11 38 03" src="https://user-images.githubusercontent.com/75235/80090125-ec845a80-8556-11ea-85fb-72b21f690f4a.png">